skimage/metrics/tests/test_set_metrics.py

```diff
@@ -114,19 +136,26 @@ def test_gallery():
     # Should return 10, the distance between the furthest tip of the kite and
     # its closest point on the diamond, which is the furthest someone can make
     # you travel to encounter your nearest neighboring point on the other set.
-    assert_almost_equal(hausdorff_distance(coords_a, coords_b), 10.)
+    assert_almost_equal(hausdorff_distance(coords_a, coords_b), 10.0)
 
     # There are two pairs of points ((30, 20), (30, 10) or (30, 40), (30, 50)),
     # that are Hausdorff distance apart. This tests for either of them.
     hd_points = hausdorff_pair(coords_a, coords_b)
-    assert np.equal(hd_points, ((30, 20), (30, 10))).all() or \
-           np.equal(hd_points, ((30, 40), (30, 50))).all()
+    assert (
+        np.equal(hd_points, ((30, 20), (30, 10))).all()
+        or np.equal(hd_points, ((30, 40), (30, 50))).all()
+    )
+
+    # Test the Modified Hausdorff function on the coordinates
+    # Should return 7.5.
+    assert_almost_equal(hausdorff_distance(coords_a, coords_b, method="modified"), 7.5)
 
 
 @pytest.mark.parametrize(
```

Comment:
You can even refactor this as
```python
@pytest.mark.parametrize("points_a", [(0, 0, 1), (0, 1, 0), (1, 0, 0)])
@pytest.mark.parametrize("points_b", [(0, 0, 2), (0, 2, 0), (2, 0, 0)])
```
It avoids the import of `itertools` :slightly_smiling_face: